vm/interpreter/src/vm.rs

```diff
@@ -43,6 +43,10 @@ use crate::{fvm::ForestExternsV2, fvm3::ForestExterns as ForestExterns_v3};
 pub(crate) type ForestMachine<DB> = DefaultMachine<DB, ForestExternsV2<DB>>;
 pub(crate) type ForestMachineV3<DB> = DefaultMachine_v3<DB, ForestExterns_v3<DB>>;
 
+const SYSTEM_ADDR: Address = Address::new_id(0);
+const CRON_ADDR: Address = Address::new_id(3);
+const REWARD_ADDR: Address = Address::new_id(2);
```

Comment:
I'm not against hoisting these out of the actor interface but this is not the place for them. If you want to copy the addresses, put them in `forest_shim`.

Reply:
moved to forest_shim